### ISSUE TITLE
[BUG] [Java Service][lite]Behavior of v1 and v2 is inconsistent when PUT API is disabled in config file.

### ISSUE THREAD

**Issue by lyiyu66** [vmware/singleton#2864]
**Describe the bug**
commit: b859baa2ccc8ca15e13953b6cae563b2bf132618
Behavior of v1 and v2 is inconsistent when PUT API is disabled in config file. 

**To Reproduce**
1. Send out the PUT request with below body: https://<host id>:8090/i18n/api/v1/translation/products/Testing1/versions/1.0.0
```
{
  "requester": "string",
  "data": {
    "productName": "Testing1",
    "version": "1.0.0",
    "dataOrigin": "string",
    "pseudo": false,
    "machineTranslation": true,
    "creation": {
      "operationid": "string"
    },
    "translation": [
      {
        "component": "string",
        "locale": "string",
        "messages": {
          "additionalProp1": "string",
          "additionalProp2": "string",
          "additionalProp3": "string"
        }
      }
    ]
  }
}
```
2. See error
V1:
```
{
    "code": 404,
    "message": "Not Found"
}
```
V2:
```
{
    "response": {
        "code": 520,
        "message": "Request method 'PUT' is not supported",
        "serverTime": ""
    },
    "signature": "",
    "data": null
}
```

**Expected behavior**
The behavior should be consistent in v1 and v2 APIs.

**Comment by tigershi** [comment]
Because the v2 PUT API have same URL GET API, there occur error "Request method 'PUT' is not supported"

**Comment by lyiyu66** [comment]
The issue has been fixed with commit b47cb100af23f5d9f4b8786c18ccdcd5a604fa33, PUT api returns to 404 when the switch is set to false.
